header.scss

```diff
@@ -0,0 +1,397 @@
+                  @include fontProp(14px, 24px, $color-link-hover);
+                }
+
+                &.current {
+                  padding-bottom: 3px;
+                  display: flex;
+                  flex-direction: column;
+                  align-items: center;
+
+                  &:after {
+                    content: "";
+                    width: 40px;
+                    height: 2px;
+                    background-color: $color-link;
+                    display: block;
+                    text-align: center;
+                    transition: all 0.3s ease 0s;
+                  }
+                }
+              }
+
+
+              &.get-skout-btn, &.login-btn {
+                button {
+                  margin: 197px 0 16px 0;
+                  border: 1px solid $color-link;
+                  border-radius: 20px;
+                  width: 180px;
+                  text-align: center;
+                  @include fontProp(12px, 24px, $color-link);
+                  letter-spacing: 2px;
+                  padding: 8px;
+                  text-transform: uppercase;
+                  background-color: $color-primary;
+                  font-family: 'Acme', sans-serif;
+
+                  &:hover {
+                    @include fontProp(14px, 24px, $color-link-hover);
+                    color: $color-primary;
+                    background-color: $color-link;
+                  }
+                }
+              }
+
+              &.login-btn {
+                button {
+                  width: 90px;
+                  margin: 0 0 40px;
+                }
+              }
+            }
+          }
+
+          &.active {
+            left: 0;
+
+            &::before {
+              content: "";
+              position: fixed;
+              width: 100%;
+              top: 0;
+              left: 0;
+              height: 62px;
+              background: linear-gradient(45deg, #12C1E6, #01F973);
+              z-index: 2;
+              border: none;
+            }
+          }
+        }
+      }
+    }
+  }
+
+  .header-container-fixed {
+    position: fixed;
+    background-color: $color-primary;
+
+    .menu-icon {
+      span, &::before, &::after {
+        background-color: $color-secondary-text !important;
+      }
+    }
+
+    a {
+      color: $color-secondary-text !important;
+
+      &.current {
+        &:after {
+          background-color: $color-secondary-text !important;
+        }
+      }
+    }
+
+    .login-btn, .get-skout-btn {
+      button {
+        border-color: $color-secondary-text !important;
+        color: $color-secondary-text !important;
+
+        &:hover {
+          @include fontProp(14px, 24px, $color-primary);
+          color: $color-primary !important;
+          background-color: $color-secondary-text !important;
+        }
+      }
+    }
+  }
+
+  h1 {
+    font-family: 'Actor', sans-serif;
+    @include fontProp(40px, 56px, $color-primary);
+    width: 38%;
```

Comment:
why do you use width here and percentages? limit a heading with padding

Reply:
done